CHANGELOG.rst

```diff
@@ -8,6 +8,186 @@
 
 .. towncrier release notes start
 
+Pytest 3.3.0 (2017-11-23)
+=========================
+
+Deprecations and Removals
+-------------------------
+
+- Pytest no longer supports Python **2.6** and **3.3**. Those Python versions
```

Comment:
Hm, this sounds like this release really should be 4.0 instead...

Reply:
Perhaps, but we decided at the time to just increase a minor version instead. 

Either way I think it might be too late to change that decision now TBH.

Reply:
Fair point - I missed the original decision. Let's see how that goes...